server/users.ts

```diff
@@ -93,7 +95,16 @@ function merge(user1: User, user2: User) {
 	prevUsers.delete(user2.userid);
 	prevUsers.set(user1.userid, user2.userid);
 }
-
+function updateForcedPublic(user: User) {
```

Comment:
This doesn't clear `forcedPublic`, so `/nick`ing to a prefixed name and back will still force public.

Reply:
Thank, clearing `forcedPublic` now.